sdk/monitor/Microsoft.Azure.Management.Monitor/src/Microsoft.Azure.Management.Monitor.csproj

```diff
@@ -6,11 +6,12 @@
 	<PropertyGroup>
 		<Description>Microsoft Azure Monitor Library</Description>
 		<AssemblyName>Microsoft.Azure.Management.Monitor</AssemblyName>
-		<Version>0.25.2-preview</Version>
+		<Version>0.25.3-preview</Version>
```

Comment:
Are we ever going to get to 1.0?  Just asking.  There are large blockers to many customers using pre-release SDKs

Reply:
Not sure who it's the appropriate person to answer this.
In the case of DataCollectionRules, it's a feature of a larger product (Azure Monitor)

Reply:
Recorded Test Session added!